packages/boot/test/bootstrapTests/test-vaults-integration.ts

```diff
@@ -171,20 +175,18 @@ test('close vault', async t => {
 
   const message =
     'Offer {"brand":"[Alleged: IST brand]","value":"[1n]"} is not sufficient to pay off debt {"brand":"[Alleged: IST brand]","value":"[5025000n]"}';
-  await t.throwsAsync(
-    wd.executeOfferMaker(
-      Offers.vaults.CloseVault,
-      {
-        offerId: 'close-insufficient',
-        collateralBrandKey,
-        giveMinted: 0.000_001,
-      },
-      'open-vault',
-    ),
+
+  // does not throw, because it's in the result, but it will show up in errors
```

Comment:
this is a breaking change to the API. I don't think it's necessary. Let's discuss.